blog-post.html

```diff
@@ -4,9 +4,83 @@
     <title>Blog Post</title>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
+    <link rel="stylesheet" href="/styles/styles.css"/>
+    <link href="https://fonts.googleapis.com/css?family=Open+Sans&display=swap" rel="stylesheet">
+    <script src="https://kit.fontawesome.com/a076d05399.js"></script>
+    <script src="https://use.fontawesome.com/ca6c48c124.js"></script>
   </head>
   <body>
-    <section id="header"></section>
-    <section id="main"></section>
+    <section id="header">
+        <h1 id="page_heading"><a href="index.html">Blog Post</a></h1>
+        <h3 id="heading_subtext">The page for the home.</h3>
+        <div id="divider_container">
+          <div id="header_divider" class="divider"></div>

```

Comment:
This is a great example of what I was referring to in the over all notes. Having this take up the same width as the whole page works well.